spec/addressable/template_spec.rb

```diff
@@ -1293,6 +1323,12 @@ def self.match(name)
     end
   end
 
+  describe 'inspect' do
+    it "returns a string representation of the MatchData's state" do
+      expect(its.inspect).to match(/\A#<Addressable::Template::MatchData:0x[0-9a-f]{0,12} RESULT:{"foo"=>"ab", "bar"=>"cd"}>\z/)
```

Comment:
Line is too long. Also not clear why this is related to the rest of the pull request.